backend/config/default.yaml

```diff
@@ -1,6 +1,10 @@
+# production configuraiton
```

Comment:
I can't get the npm config library working either in CI or local env.

I have actually refactored it into using `.env` files which are written from Github Actions for all react, node and playwright projects and it seems working.

See my pr